Makefile

```diff
@@ -35,7 +35,7 @@ clean:
 $(RUNTIME_TOOLS_LINK):
 	ln -sf $(CURDIR) $(RUNTIME_TOOLS_LINK)
 
-.PHONY: test .gofmt .govet .golint
```

Comment:
Because both cases can be implemented, and I did not find an accurate description of which is wrong. Just a personal feeling that would be better.